src/main/java/com/labsynch/labseer/domain/StandardizationDryRunCompound.java

```diff
@@ -225,6 +238,24 @@ private static Predicate buildPredicatesForSearch(CriteriaBuilder criteriaBuilde
 			predicates.add(criteriaBuilder.equal(root.get("asDrawnDisplayChange"), dryRunSearch.getAsDrawnDisplayChange()));
 		}
 
+		// Standardization statuses
+		if (dryRunSearch.getStandardizationStatuses() != null) {
+			if(dryRunSearch.getStandardizationStatuses().length > 0) {
+				predicates.add(root.get("standardizationStatus").in(dryRunSearch.getStandardizationStatuses()));
+			} else {
+				predicates.add(criteriaBuilder.equal(root.get("id"), -1));
```

Comment:
This seems rather aggressive - if no statuses are selected we return zero results, rather than simply not applying a filter? Maybe this will make more sense when I see the UI.

Reply:
yeah, I suppose it is agressive and it was questionable when I did it.  My logic

 - Passing in an actual list -> Filter by list
 - Pass null -> Don't filter
 - Pass empty list ->  Return nothing because somebody might create a UI where this is a state of the UI and expect 0 results

Reply:
FWIW, I did try do a in an empty list but it wasn't working for me so I did this instead.

Reply:
OK. This will go through functional testing so I'm alright with this.